conf/nginx.conf

```
@@ -5,21 +5,22 @@ location __PATH__/ {
         rewrite ^ https://$server_name$request_uri? permanent;
     }
 
-    # This is important for user's privacy !
+    # This is important for user's privacy!
     access_log off;
     error_log /var/log/nginx/lufi.error.log;
 
-    # This is important ! Make it OK with your Lutim configuration
+    # This is important! Make it OK with your Lutim configuration
     client_max_body_size __MAX_FILE_SIZE__M;
 
     if ($request_uri ~* ^/(img|css|font|js)/) {
-         more_set_headers Expires "Thu, 31 Dec 2037 23:55:55 GMT";
-         more_set_headers Cache-Control "public, max-age=315360000";
+         # If uncommented, it triggers some net::ERR_HTTP2_PROTOCOL_ERROR with chromium
+         #more_set_headers Expires "Thu, 31 Dec 2037 23:55:55 GMT";
+         #more_set_headers Cache-Control "public, max-age=315360000";
```

Isnt it a syntax issue ? Shouldn't this be : 

```
```suggestion
         more_set_headers "Expires: Thu, 31 Dec 2037 23:55:55 GMT";
         more_set_headers "Cache-Control: public, max-age=315360000";
```

(the syntax is different than the regular `add_header`, so yeah that's a pain in the ass

---

(Ping @zamentur since you're the one that committed this)